shared/js/smart-screen/scrollable.js

```diff
@@ -459,6 +471,63 @@
       return true;
     },
 
+    hover: function(item1, item2) {
+      var node1 = this.getNodeFromItem(item1);
+      var node2 = this.getNodeFromItem(item2);
+
+      if (!node1 || !node2) {
+        return false;
+      }
+
+      var idx1 = parseInt(node1.dataset.idx, 10);
+      var idx2 = parseInt(node2.dataset.idx, 10);
+
+      node1.style.transform = 'translateX(calc((100% + ' +
+                            this.spacing + 'rem) * ' + (idx1 + idx2) / 2 + '))';
+
+      node2.style.transform = 'translateX(calc((100% + ' +
+                            this.spacing + 'rem) * ' + (idx1 + idx2) / 2 + '))';
+
+      this.fire('hover', this);
+      this.isHovering = true;
+      item1.classList.add('hover');
+      item2.classList.add('hovered');
+      node1.classList.add('hover');
+      node2.classList.add('hovered');
```

Comment:
It's ok to leave them inside though; since removing them involves some internal cases of `scrollable`. The reason I asked this is because `focus` is added/removed outside.